README.md

```diff
@@ -503,11 +503,12 @@ of a breaking change with these two widely used fields in Beats.
 Any future field that will be indexed for full text search in ECS will however
 follow the multi-field convention where `text` indexing is nested in the multi-field.
 
-### IDs are keywords not integers
+### IDs and codes are keywords, not integers
```

Comment:
Are you thinking of some specific fields here?

Reply:
Yes, error codes. Came up during a discussion with @graphaelli over on #154